joomla/joomla-cms
# Refactor com_postinstall from FOF to Joomla Core

#### Summary of Changes
- Refactor com_postinstall from FOF to Joomla core
- Remove dependencies to FOF
#### Testing Instructions
- Step 1: Go to the backend 
- Step 2: Navigate to Components->Post installation Messages
- Step 3: All available post install messages should be shown
- Step 4: Clicking the "Hide this message" button should hide the message
- Step 5: Clicking a blue action button will execute appropriated action
  (e.g. 2FA Message action button will activate the 2FA plugin)
- Step 6: If all messages are hidden a reset button will appear, clicking it will reset all messages
- Step 7: Clicking the "Help" Button in the toolbar will popup a help window
- Step 8: Clicking the "Options" Button in the toolbar will lead you to the component options
- Step 9: After applying the changes Step 1-8 should work the same.
